### PR TITLE
Update to add R 4.0.0

### PR DESCRIPTION
**Pull request by jkwmoore**
Update to add R 4.0.0 info - this is still in staging at present! Await testing before merging.

### CODE REVIEW

**Comment by jkwmoore** [comment]
Associated issue here: #1110

Awaiting testing before moving build out of staging and merging this into the docs.

File: bessemer/software/apps/R.rst [hunk 1]
```diff
@@ -168,6 +172,14 @@ Installation Notes
 ------------------
 These notes are primarily for administrators of the system.
 
+R/4.0.0-foss-2020a
+^^^^^^^^^^^^^^^^^^
+
+Installed using an eponymous easyconfig,
+which is the easyconfig that shipped with EasyBuild 4.2.2
```
Comment (willfurnass) [inline bessemer/software/apps/R.rst:179]: Used 4.2.2 here?

Comment (jkwmoore) [reply bessemer/software/apps/R.rst:179]: Good catch - Edited this elsewhere and missed it here! Amending.

**Comment by willfurnass** [comment]
@jkwmoore Ready to review?

**Comment by jkwmoore** [comment]
> 
> 
> @jkwmoore Ready to review?

Should be good to go iirc.

**Pull request merged**